docs/pdoc_templates/menu.mako

```diff
@@ -1,142 +1,494 @@
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="/guide/">
+            Docs
+            <svg aria-hidden="true" width="10" height="6" viewBox="0 0 10 6" fill="none" xmlns="http://www.w3.org/2000/svg" class="page-header-chevron-icon"><path d="M1 0.5L5 4.5L9 0.5" stroke="#131522"></path></svg>
+          </a>
+          <button>
+            <svg width="13" height="8" viewBox="0 0 13 8" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+              <path fill-rule="evenodd" clip-rule="evenodd" d="M0.292969 0.805007L1.00008 0.0979004L6.64652 5.74435L12.293 0.0979004L13.0001 0.805007L6.64652 7.15856L0.292969 0.805007Z"></path>
+            </svg>
+          </button>
+          <ul>
+            <li>
+              <a href="/guide/index.html#Quick-start">Quickstart</a>
+            </li>
+            <li>
+              <a href="/guide/install.html">Install</a>
+            </li>
+            <li>
+              <a href="/guide/tasks.html">Import Data</a>
+            </li>
+            <li>
+              <a href="/api">API Reference</a>
+            </li>
+            <li>
+              <a href="/sdk">SDK Reference</a>
+            </li>
+            <li class="headerSeparator">
+              <a href="/tags">Customizable Tags</a>
+            </li>
+            <li>
+              <a href="/templates">Labeling Templates</a>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="/community/">
+            Community
+          </a>
+        </li>
+        <li>
+          <a href="https://heartex.com/free-trial" target="_blank" rel="noopener noreferrer">Try Cloud</a>
+        </li>
+        <li>
+          <a href="https://github.com/heartexlabs/label-studio/" class="github-stars" target="_blank" rel="noopener noreferrer">
+            <svg class="github-icon" viewBox="0 0 24 24" width="24" height="24"><path fill="black" d="M12 2A10 10 0 0 0 2 12c0 4.42 2.87 8.17 6.84 9.5.5.08.66-.23.66-.5v-1.69c-2.77.6-3.36-1.34-3.36-1.34-.46-1.16-1.11-1.47-1.11-1.47-.91-.62.07-.6.07-.6 1 .07 1.53 1.03 1.53 1.03.87 1.52 2.34 1.07 2.91.83.09-.65.35-1.09.63-1.34-2.22-.25-4.55-1.11-4.55-4.92 0-1.11.38-2 1.03-2.71-.1-.25-.45-1.29.1-2.64 0 0 .84-.27 2.75 1.02.79-.22 1.65-.33 2.5-.33.85 0 1.71.11 2.5.33 1.91-1.29 2.75-1.02 2.75-1.02.55 1.35.2 2.39.1 2.64.65.71 1.03 1.6 1.03 2.71 0 3.82-2.34 4.66-4.57 4.91.36.31.69.92.69 1.85V21c0 .27.16.59.67.5C19.14 20.16 22 16.42 22 12A10 10 0 0 0 12 2z"></path></svg>
+            <span class="Text github-stars-count Eyebrow ">11,337</span>
+          </a>
+        </li>
+        <li>
+          <a href="https://labelstud.io/guide/index.html#Quick-start" class="Button">
+            Quick start
+          </a>
+        </li>
+    </ul>
+  </nav>
+</header>
 
 <script src="/js/css.escape.js"></script>
 <script src="/js/common.js"></script>
-
+<script src="https://label-studio-docs-new-theme.netlify.app/js/header.js" />
```

Comment:
`https://label-studio-docs-new-theme.netlify.app` is just for review purposes. I’ll remove it before we merge this.